src/site/pages/digital/src/state/ICViewer.ts

```diff
@@ -5,15 +5,15 @@ import {CreateState} from "shared/utils/CreateState";
 const [initialState, actions, reducer] = CreateState()(
     {
         isActive: false,
-        ic: undefined as ICData,
+        ic: undefined as unknown as ICData,
```

Comment:
^

Reply:
There are some weird errors coming up as a result of this. The error message seems kind of long, I think you will need to check out the branch and look at it yourself (or we screenshare at some point)